From: Stefano Rivera <stefanor@debian.org>
Date: Thu, 28 Jan 2021 22:15:32 -0700
Subject: Drop pytest.py filename check from test_general

Upstream was pinning an old pytest version, but have dropped the pin in
https://github.com/getsentry/sentry-python/commit/38b983e490ad4bda8db7a80ee52cfb65c398a45c#diff-a9b641cc8f69df0cae0be6d2bb7178e9629fe674557444a0f0dd61484a52a52e

Forwarded: not-necessary
---
 tests/utils/test_general.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/utils/test_general.py b/tests/utils/test_general.py
index 71cb342..e25e378 100644
--- a/tests/utils/test_general.py
+++ b/tests/utils/test_general.py
@@ -63,7 +63,6 @@ def test_filename():
     assert x("bogus", "bogus") == "bogus"
 
     assert x("os", os.__file__) == "os.py"
-    assert x("pytest", pytest.__file__) == "pytest.py"
 
     import sentry_sdk.utils
 
